src/doc/imagebufalgo.rst

```diff
@@ -2926,32 +2926,20 @@ Import / export
   Examples:
 
     .. tabs::
-  
-       .. code-tab:: c++
-
-          ImageBuf Input ("in.exr");
-          ImageSpec config;
-          config["maketx:highlightcomp"] = 1;
-          config["maketx:filtername"] = "lanczos3";
-          config["maketx:opaque_detect"] = 1;
-      
-          bool ok = ImageBufAlgo::make_texture (ImageBufAlgo::MakeTxTexture,
-                                                Input, "texture.exr", config);
-          if (! ok)
-              std::cout << "make_texture error: " << OIIO::geterror() << "\n";
 
-       .. code-tab:: py
+         .. tab:: C++
+             .. literalinclude:: ../../testsuite/docs-examples-cpp/src/docs-examples-imagebufalgo.cpp
+              :language: c++
+              :start-after: BEGIN-imagebufalgo-make-texture
+              :end-before: END-imagebufalgo-make-texture
+              :dedent: 4
```

Comment:
It turns out that you can't have a mix of `tab` and `code-tab`. It looks ok at first, but things go wrong when you switch among the tabs. I think that in order to look right, this is how you want this section (minor edits):

```
  .. tabs::

     .. tab:: C++
           .. literalinclude:: ../../testsuite/docs-examples-cpp/src/docs-examples-imagebufalgo.cpp
            :language: c++
            :start-after: BEGIN-imagebufalgo-make-texture
            :end-before: END-imagebufalgo-make-texture
            :dedent: 4

     .. tab:: Python
        .. literalinclude:: ../../testsuite/docs-examples-python/src/docs-examples-imagebufalgo.py
            :language: py
            :start-after: BEGIN-imagebufalgo-make-texture
            :end-before: END-imagebufalgo-make-texture
            :dedent: 4

     .. tab:: oiiotool
        .. sourcecode::

          oiiotool in.exr -otex:hilightcomp=1:filtername=lanczos3:opaque_detect=1 texture.exr

     .. tab:: maketx
        .. sourcecode::

          maketx in.exr --hicomp --filter lanczos3 --opaque-detect -o texture.exr
```

Reply:
Ah - that makes sense, I was actually building the docs too and testing that these tabs works and sometimes some would disappear on me.
Let me test with this more consistent way and see if that sticks more robustly.

For a second there I was thinking of maybe trying to add yet another test that tests the bash (shell) commands, but it was tricky to get to a clean "oiiotool" command that'll work across all platforms in a bash script.
It would be neat if we could do that because that would make the doc consistent across all the tabs (and provide some test coverage for the maketx/oiio flags).

Reply:
Yeah, there are some kinks still to work out with the best way to do this kind of think for the oiiotool examples, but we'll come back to that later.

Reply:
Tried your suggestion out, it does look more stable now when I test the docs, I see we have a few examples in there already that mix tab/codetab (e.g imagebufalgo-zero).
So there is a bit more cleanup to do, but we can do that in a separate PR